.github/workflows/test.yml

```diff
@@ -35,10 +38,16 @@ jobs:
           key: ${{ runner.os }}-${{ steps.nvm.outputs.NVMRC }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-${{ steps.nvm.outputs.NVMRC }}-nodemodules-
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```

Comment:
this might get deprecated in the future -> use `foundry-rs/foundry-toolchain@v1`